docs/01-index.md

```diff
@@ -1,29 +1,23 @@
 <img src="{{ path '/images/design-cover-img@2x.jpg' }}" alt="Welcome to design.stjude.cloud!" width="1280px" height="320px" style="max-width: unset"/>
 
 ## Getting Started
```

Comment:
I'd like to provide a little more distinction to content in this section as it now has both design and development. Maybe adding section headers for both?

Reply:
Added separate headers for development and the getting started.